Tribler/Test/test_as_server.py

```diff
@@ -465,6 +477,8 @@ def callLater(self, seconds, callback):
                 callback()
 
     def quit(self):
+        # Martijn: stop wx thread watchdog event sooner to prevent errors when yielding in the GUI shutdown
+        self.watchdog.unregister_event("wx thread")
```

Comment:
This means that if something really freezes while shutting down, we will never see the stacktrace to help us debug it.